Support screen share feature from client-js 0.3.2 (#10)
pipecat-ai/pipecat-client-react-native-daily-transport: README.md

await voiceClient.start()
```

> If you wish to enable screen sharing on iOS or Android, you must first follow the instructions outlined in the Daily Framework RN Screen Share extension [here](https://github.com/daily-co/rn-screen-share-extension/).

Reviewer: Nit: Just leave it as a note so we can highlight this.

```suggestion
> Note: To enable screen sharing on iOS, follow the instructions in the [Daily Framework RN Screen Share extension](https://github.com/daily-co/rn-screen-share-extension/).
```
